TRexLib.Tests/ParserTests.cs

```diff
@@ -101,6 +102,11 @@ public void Test_fail_outcomes_are_read_correctly()
         [Fact]
         public void The_path_to_a_Windows_test_project_is_parsed()
         {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return;
```

Comment:
It is actually failing. Let me dig more

Reply:
It is FileInfo behave differently on windows vs mac.

```
 Expected string to be
"C:\dev\github\cli\test\Microsoft.DotNet.Cli.Sln.Internal.Tests\bin\Debug\netcoreapp1.0\Microsoft.DotNet.Cli.Sln.Internal.Tests.dll" with a length of 130, but
"/Users/williamli/Documents/t-rex/TRexLib.Tests/bin/Debug/netcoreapp2.1/C:\dev\github\cli\test\Microsoft.DotNet.Cli.Sln.Internal.Tests\bin\Debug\netcoreapp1.0\Microsoft.DotNet.Cli.Sln.Internal.Tests.dll"
```

i am not sure it is a bug.

Reply:
I think it's a valid scenario because I might pull TRX files off of a CI server and t-rex them on my dev box running a different OS. Also, it looks like these tests were passing before.

Reply:
really? ci has only windows tho

fileinfo.fullname changed behavior? but anyhow, fullname is different per OS. we need something like `fileinfo.fullnameAsWindows`? But I don't think there is an api like that, maybe we just don't show full name if OS mismatches, use the relative path?

Reply:
(i'm in mvp hackathon so i have all the right working on this...)

I just realize getting full name is just wrong. Once you moved the trx, the relative path will not be correct anymore. If we think download TRX and run trex is common, we should just display the path as is